specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json

```diff
@@ -123,6 +123,78 @@
         }
       }
     },
+    "/{resourceId}/query": {
```

Comment:
does it mean that the URL will be created with double "/"? I'll assume it should be fine by Draft service but just wanted to make sure..

Reply:
it should be fine - but removed the '/' for the sake of consistency

Reply:
This is very much a valid question, `//foo/query` != `/foo/query` unless the service has decided to treat them the same.